### PR TITLE
aapt, ab: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages.fr/common/aapt.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+> Android Asset Packaging Tool.
+> Compile et empaquette les ressources d'une application Android.
+> Plus d'informations : <https://elinux.org/Android_aapt>.
+
+- Liste les fichiers contenus une archive APK :
+
+`aapt list {{chemin/vers/app.apk}}`
+
+- Affiche les metadatas d'une application (version, permissions, etc.) :
```
Comment (EmilySeville7cfg) [inline pages.fr/common/aapt.md:11]:
```suggestion
- Affiche les metadatas d'une application (version, autorisations, etc.):
```

Google translate suggests this translation. :)

File: pages.fr/common/ab.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# ab
+
+> Outil d'analyse pour serveur Apache HTTP.
+> Plus d'informations : <https://httpd.apache.org/docs/current/programs/ab.html>.
+
+- Exécute 100 requêtes HTTP GET sur une URL donnée :
```
Comment (EmilySeville7cfg) [inline pages.fr/common/ab.md:6]:
```suggestion
- Exécute 100 requêtes HTTP GET sur une URL donnée:
```

File: pages.fr/common/ab.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`ab -n {{100}} {{url}}`
+
+- Exécute 100 requêtes HTTP GET en parallèle par groupe de 10, sur une URL :
```
Comment (EmilySeville7cfg) [inline pages.fr/common/ab.md:10]:
```suggestion
- Exécute 100 requêtes HTTP GET en parallèle par groupe de 10, sur une URL:
```

File: pages.fr/common/ab.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`ab -n {{100}} -c {{10}} {{url}}`
+
+- Exécute 100 requêtes HTTP POST sur une URL, en utilisant un contenu JSON depuis un fichier :
```
Comment (EmilySeville7cfg) [inline pages.fr/common/ab.md:14]:
```suggestion
- Exécute 100 requêtes HTTP POST sur une URL, en utilisant un contenu JSON depuis un fichier:
```

File: pages.fr/common/ab.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`ab -n {{100}} -T {{application/json}} -p {{chemin/vers/le/fichier.json}} {{url}}`
+
+- Utilise la fonctionalitée HTTP Keep Alive pour exécuter plusieurs requêtes dans la même session HTTP :
```
Comment (EmilySeville7cfg) [inline pages.fr/common/ab.md:18]:
```suggestion
- Utilise la fonctionalitée HTTP Keep Alive pour exécuter plusieurs requêtes dans la même session HTTP:
```

File: pages.fr/common/ab.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`ab -k {{url}}`
+
+- Fixe le nombre maximum de secondes d'exécution pour l'analyse :
```
Comment (EmilySeville7cfg) [inline pages.fr/common/ab.md:22]:
```suggestion
- Fixe le nombre maximum de secondes d'exécution pour l'analyse:
```

File: pages.fr/common/aapt.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+`aapt dump badging {{chemin/vers/app.apk}}`
+
+- Créé une nouvelle archive APK avec les fichiers venant d'un dossier spécifique :
```
Comment (EmilySeville7cfg) [inline pages.fr/common/aapt.md:15]:
```suggestion
- Créé une nouvelle archive APK avec les fichiers venant d'un dossier spécifique:
```

File: pages.fr/common/aapt.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
+# aapt
+
+> Android Asset Packaging Tool.
+> Compile et empaquette les ressources d'une application Android.
+> Plus d'informations : <https://elinux.org/Android_aapt>.
+
+- Liste les fichiers contenus une archive APK :
```
Comment (EmilySeville7cfg) [inline pages.fr/common/aapt.md:7]:
```suggestion
- Liste les fichiers contenus une archive APK:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages.fr/common/ab.md (hunk 2 above)
Comment (navarroaxel) [reply pages.fr/common/ab.md:6]: That space is a specific French rule.

File: pages.fr/common/aapt.md (hunk 1 above)
Comment (Nico385412) [reply pages.fr/common/aapt.md:11]: both are synonyms and work in that context ^^
So if you prefer `autorisation` no problem 😉

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Thanks for your contribution. 😺

**Pull request merged**